src/github/logic.py

```diff
@@ -201,7 +202,9 @@ def all_pull_request_participants(pull_request: PullRequest) -> List[str]:
                 [pull_request.author_handle()]
                 + pull_request.assignees()
                 + pull_request.reviewers()
-                + pull_request.requested_reviewers()
+                + pull_request.requested_reviewers(
+                    include_team_members=not SGTM_FEATURE__DISABLE_GITHUB_TEAM_SUBSCRIPTION
+                )
                 + _pull_request_commenters(pull_request)
```

Comment:
Unfortunately, I think you're going to run into a merge conflict with #143  - since that change is a bit more involved, do you mind re-implementing this change on top of that one? I'm expecting to get it merged tomorrow when @suzyng83209 is back